### PR TITLE
CS: Alter user table task complete. Email, first_name & surname added…

### PR DESCRIPTION
**Pull request by CharlesMarcusSmith**
CS: Alter user table task complete. Email, first_name & surname added as file type 'TEXT'.

### CODE REVIEW

**Comment by CharlesMarcusSmith** [comment]
This is migration V4, however creation of the whale spotting table is migration V3, so this might need to be considered when merging?

**Review by charlottemothersole — COMMENTED** [review]
All good!

**Review by charlottemothersole — COMMENTED** [review]
Following tim's explanation you might want to add in an extra line at the bottom of the file? Otherwise good to merge!

File: src/main/resources/db/migration/V4__alter_user_table.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ALTER TABLE "member"
+ADD COLUMN email TEXT,
+ADD COLUMN first_name TEXT,
+ADD COLUMN surname TEXT;
```
Comment (TimLeach635) [inline src/main/resources/db/migration/V4__alter_user_table.sql]: Blank line at end of file
```suggestion
ADD COLUMN surname TEXT;
```

**Review by TimLeach635 — DISMISSED** [review]

**Comment by CharlesMarcusSmith** [comment]
This is still waiting to be approved, line has been added.

**Review by AndrewYHuang — APPROVED** [review]
Looks good to me!

**Comment by CharlesMarcusSmith** [comment]
Merging is still blocked

**Pull request merged**